nats_events/nats_events/v1_0/nats_queue/config.py

```diff
@@ -0,0 +1,164 @@
+"""NATS Queue configuration."""
+
+import logging
+from typing import Any, Mapping, Optional
+
+from pydantic import BaseModel
+
+LOGGER = logging.getLogger(__name__)
+
+EVENT_TOPIC_MAP = {
+    "^acapy::webhook::(.*)$": "acapy-webhook-$wallet_id",
+    "^acapy::record::([^:]*)::([^:]*)$": "acapy-record-with-state-$wallet_id",
+    "^acapy::record::([^:])?": "acapy-record-$wallet_id",
+    "acapy::basicmessage::received": "acapy-basicmessage-received",
+    "acapy::problem_report": "acapy-problem_report",
+    "acapy::ping::received": "acapy-ping-received",
+    "acapy::ping::response_received": "acapy-ping-response_received",
+    "acapy::actionmenu::received": "acapy-actionmenu-received",
+    "acapy::actionmenu::get-active-menu": "acapy-actionmenu-get-active-menu",
+    "acapy::actionmenu::perform-menu-action": "acapy-actionmenu-perform-menu-action",
+    "acapy::keylist::updated": "acapy-keylist-updated",
+    "acapy::revocation-notification::received": "acapy-revocation-notification-received",
+    "acapy::revocation-notification-v2::received": "acapy-revocation-notification-v2-received",  # noqa: E501
+    "acapy::forward::received": "acapy-forward-received",
+    "acapy::outbound-message::queued_for_delivery": "acapy-outbound-message-queued-for-delivery",  # noqa: E501
+}
+
+EVENT_WEBHOOK_TOPIC_MAP = {
+    "acapy::basicmessage::received": "basicmessages",

```

Comment:
could be useful for name spacing to keep the prefixes, e.g. `acapy.basicmessage` and potentially the suffix as well to future proof in case there are other basicmessage sub contexts introduced in the future e.g. `acapy.basicmessage.received`

Reply:
These mappings are only used for dispatching webhooks to custom urls (which we have disabled)